source/windows/windows_pki_utils.c

```diff
@@ -640,6 +640,8 @@ int aws_import_key_pair_to_cert_context(
 
     struct aws_pem_object *private_key_ptr = NULL;
     DWORD decoded_len = 0;
+    BYTE *key_wrapper = NULL;
```

Comment:
I think we clean up key with LocalFree(key) down below. Do we know whether we need to also free this the same way whether in the event of a success or failure?

Reply:
Docs says nothing about this. I checked locally - `key` is NULL in case of failure (which is expected). `LocalFree` does nothing when its parameter is NULL, so it's safe to call it after all attempts to decode the key.
But I decided to move `LocalFree(key_wrapper)` to the clean_up section, so if we add another case with key_wrapper, its destroying will be already handled.